app/jobs/generate_ptiff_job.rb

```diff
@@ -18,7 +18,7 @@ def perform(child_object, batch_process)
     # TODO: Uncomment the line below to re-implement Goobi ingest and copy to the access master pair-tree.
     # There is a test in spec/models/batch_process_spec.rb that should fail, take out the "pending" and logger
     # mocks there to see if it passes
-    child_object.copy_to_access_master_pairtree if child_object.parent_object.from_mets
+    child_object.copy_to_access_master_pairtree if child_object.parent_object.from_mets || child_object.preservica_bitstream_uri
```

Comment:
`copy_to_access_master_pairtree` isn't necessary for preservica items, since it happens earlier in `preservica_copy_to_access`